script/Ciel/Day 13/13_01_01_CIEL13_3.txt

```diff
@@ -66,17 +67,17 @@ A recreational area built in the heart of the business district.
 [sha:65c60eca888ee07e7724d92d7e7e890729b1bb53]{
 -- Page 104, Offset 33460.
 -- 日中は人の姿が見られるが、夜になれば<人気|ひとけ>の絶える都市の死角。
-You would see people here during the day, but at night, it becomes the blind spot of the city, devoid of life.
+It's populated during the day, but when the night falls it bcomes a deserted blind spot of the city.
 }
 [sha:997c077afd14ca34d67db1cd947f582a4633e663]{
 -- Page 105, Offset 33461.
 -- 墓地のように静まりかえった公園は、しかし、人の気配で溢れていた。
-The park that's silent as a graveyard, however, was brimming with life signals.
+Even though the park itself is as silent as a graveyard, it's still brimming with signs of life.
 }
 [sha:5525120bf401fc1e586fc5565ce5271d8ea69ec0]{
 -- Page 105, Offset 33462.
 -- 誰かの息遣いか、耳障りな話し声が、真綿のように全身を締め上げる。
-The peoples' breathing, and their grating talking voices, tighten around my entire body like floss silk.
+Their breathing and their grating voices are tightening around my body like silk.
```

Comment:
I'm guessing this is like 真綿で首を絞める rather than literally being tightened with silk.

Their breathing and their grating voices are slowly breaking me down.